srtcore/core.cpp

```diff
@@ -1862,8 +1862,10 @@ size_t CUDT::fillHsExtGroup(uint32_t* pcmdspec)
     if (m_parent->m_IncludedGroup->synconmsgno())
         flags |= SRT_GFLAG_SYNCONMSG;
 
+#if ENABLE_HEAVY_LOGGING
+
     SRTSOCKET master_peerid;
-    IF_HEAVY_LOGGING(steady_clock::duration master_tdiff);
+    steady_clock::duration master_tdiff;
```

Comment:
`master_tdiff can be local (moved inside further ifs so that it is not visible in the whole function).